tests/kafkatest/tests/streams/streams_broker_bounce_test.py

```diff
@@ -112,6 +112,8 @@ def __init__(self, test_context):
             'wcnt' : { 'partitions': self.partitions, 'replication-factor': self.replication,
                        'configs': {"min.insync.replicas": 2} },
             'tagg' : { 'partitions': self.partitions, 'replication-factor': self.replication,
+                       'configs': {"min.insync.replicas": 2} },
+            '__consumer_offsets' : { 'partitions': 50, 'replication-factor': self.replication,
```

Comment:
This is intentional to avoid streams block on starting up trying to find the coordinator because the offset topic cannot be created with replication factor 3.